theseus_gui/src/pages/Index.vue

```diff
@@ -0,0 +1,32 @@
+<script setup>
+import { useInstances, useNews } from '@/store/state'
+import RowDisplay from '@/components/RowDisplay.vue'
+
+const instances = useInstances()
+const news = useNews()
+instances.fetchInstances()
+news.fetchNews()
+
+const recentInstances = instances.instances.filter((i) => i.downloads <= 50)
+const popularInstances = instances.instances.filter((i) => i.downloads > 50)
+const trendingMods = instances.instances.filter((i) => i.trending)
+</script>
+
+<template>
+  <div class="page-container">
+    <RowDisplay label="Jump back in" :instances="recentInstances" />
+    <RowDisplay label="Popular packs" :instances="popularInstances" />
+    <RowDisplay label="News & updates" :news="news.news" />
+    <RowDisplay label="Trending mods" :instances="trendingMods" />
```

Comment:
The trending mods section is styled very weirdly. It's layout should be styled similar to how we style them on our homepage: 
![image](https://user-images.githubusercontent.com/18202329/224118424-470cd2eb-ebd0-4e46-937a-f95cb4daede3.png)